apps/shared/avifutil.c

```diff
@@ -248,7 +251,8 @@ avifAppFileFormat avifReadImage(const char * filename,
             *outDepth = 8;
         }
     } else if (format == AVIF_APP_FILE_FORMAT_PNG) {
-        if (!avifPNGRead(filename, image, requestedFormat, requestedDepth, outDepth)) {
+        (void)ignoreICC, (void)ignoreXMP; // TODO(yguyon): Implement
```

Comment:
FYI, recent PNG spec extension 1.6 also added a cICp chunk, so for now you just might want to add it to the feature TODO list here (and also the force/ignore options). There is no libpng API for it yet, so a "user chunk" API can be used. So far, I think only libjxl started outputting PNGs like these when transcoding...

https://w3c.github.io/PNG-spec/

Reply:
Good to know.